src/App.tsx

```diff
@@ -2,13 +2,21 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostType } from './types/PostType';
+import { PostInfo } from './components/PostInfo';
+
+export const preparedPosts = posts.map<PostType>(post => ({
+  ...post,
+  user: users.find(person => person.id === post.userId),
+  comments: comments.filter(com => post.id === com.postId),
+}));
 
 const App: React.FC = () => (
   <div className="App">
-    <h1>Static list of posts</h1>
+    {preparedPosts.map(post => <PostInfo {...post} />)}
```

Comment:
there should be `key` attribute

Reply:
fixed now, I added it in the PostInfo div